File: 1 - BEGINNER/1012 - Área.c
#include <stdio.h>
 
int main() {
 
    double a, b, c, triangulo, circulo, trapezio, quadrado, retangulo, pi = 3.14159;
    
    scanf("%lf %lf %lf", &a, &b, &c);
    
    triangulo = a * c / 2.0; //A área do triângulo retângulo é dada pelo produto da sua base pela sua altura, tudo isso dividido por 2, já que ele é a metade de um retângulo com as mesmas dimensões de base e altura, ou seja: base * altura / 2
    circulo = pi * c * c; //A área do círculo é dada por pi = 3.14159... multiplicado pelo quadrado de seu raio, ou seja: pi * raio * raio
    trapezio = (a + b) * c / 2.0; //A área do trapézio é dada pela soma de sua base menor com sua base maior, multiplicada por sua altura, tudo isso dividido por 2, ou seja: [(base maior + base menor) * altura] / 2
    quadrado = b * b; //A área de um quadrado é dada pelo quadrado de seu lado, ou seja: lado * lado
    retangulo = a * b; //A área de um retângulo é dada pela multiplicação de seu lado por sua altura, ou seja: lado * altura
    //obs: caso tenha dificuldades em relação a isso, tente resolver após estudar mais sobre geometria plana.
    printf("TRIANGULO: %.3lf\n", triangulo);
    printf("CIRCULO: %.3lf\n", circulo);
    printf("TRAPEZIO: %.3lf\n", trapezio);
    printf("QUADRADO: %.3lf\n", quadrado);
    printf("RETANGULO: %.3lf\n", retangulo);
 
    return 0;
}

File: 1 - BEGINNER/Java/1012 - Área.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    
    public static final double PI = 3.14159;
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        double A = input.nextDouble();
        double B = input.nextDouble();
        double C = input.nextDouble();
        double triangulo = A * C / 2.0;
        double circulo = PI * C * C;
        double trapezio = (A + B) * C / 2.0;
        double quadrado = B * B;
        double retangulo =  A * B;
        
        System.out.printf("TRIANGULO: %.3f\n", triangulo);
        System.out.printf("CIRCULO: %.3f\n", circulo);
        System.out.printf("TRAPEZIO: %.3f\n", trapezio);
        System.out.printf("QUADRADO: %.3f\n", quadrado);
        System.out.printf("RETANGULO: %.3f\n", retangulo);
    
    }
 
}